.travis.yml

```diff
@@ -1,6 +1,6 @@
-language: c
-dist: trusty # because of perf issues
-sudo: required
+language: node_js
+node_js:
+  - "10"
```

Comment:
Did you want 10 or just the latest version?

```
language: node_js
  - "node"
  # ^ use latest version of NodeJS
```

Reply:
I want "10" because I want to opt in to upgrades later on, I don't want them to happen automatically.

Reply:
Makes sense.